testdata/p4_16_samples_outputs/parser-locals-midend.p4

```diff
@@ -14,7 +14,6 @@ struct S {
 parser p() {
     H s_0_h1;
     H s_0_h2;
-    bit<32> s_0_c;
```

Comment:
this is weird, the code had an unused variable?

Reply:
They're all dead and could be removed; the two headers are not due to the (implicit) `setInvalid` calls to initialize them.  That's because in `copyprop_name` when dealing with arguments to a method call, we assume that all `out` and `inout` args are implicitly read, which is not generally true.